README.md

```diff
@@ -215,3 +215,9 @@ Please see the [CONTRIBUTING.md][2]. For security bugs and vulnerabilities, plea
 [1]: https://github.com/valkey-io/valkey/blob/unstable/COPYING
 [2]: https://github.com/valkey-io/valkey/blob/unstable/CONTRIBUTING.md
 [3]: https://github.com/valkey-io/valkey/blob/unstable/SECURITY.md
+
+Valkey is a Linux Foundation project
```

Comment:
@mkdolan You okay with this wording? "About Valkey" seems a bit weird since this whole document is about valkey.

We maybe should also add it here: https://valkey.io/about/

Reply:
How about "Valkey is an open community project under LF Projects"

This addition to the page is for lawyers, so I get it may seem "awkward" but it would be useful to have at the bottom.

Reply:
This is fine to also include on the website page.